2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -37,8 +41,10 @@ function cache_container_addr() {
 
 # wait_for_mongo_up waits until the mongo server accepts incomming connections
 function wait_for_mongo_up() {
-  local mongo_host="${1-}"
-  local mongo_cmd="mongo admin "
+  local mongo_host
+  mongo_host="${1-}"
```

Comment:
It is explained here: https://google.github.io/styleguide/shell.xml?showone=Use_Local_Variables#Use_Local_Variables